content/docs/3.0/faq/faq.md

```diff
@@ -0,0 +1,151 @@
+
+* Atomicity: Each transaction in TiKV is "all or nothing": if one part of the transaction fails, then the entire transaction fails, and the database state is left unchanged. TiKV guarantees atomicity in each and every situation, including power failures, errors, and crashes. 
+* Consistency: TiKV ensures that any transaction brings the database from one valid state to another. Any data written to the TiKV database must be valid according to all defined rules, including constraints, cascades, triggers, and any combination thereof. 
+* Isolation: TiKV provides snapshot isolation (SI), snapshot isolation with lock (SQL statement: SELECT ... FOR UPDATE), and externally consistent reads and writes in distributed transactions. 
+* Durability: TiKV allows a collection of machines to work as a coherent group that can survive the failures of some of its members. So in TiKV, once a transaction has been committed, it will remain so, even in the event of power loss, crashes, or errors. 
+
+## How are transactions in TiKV lock-free?
+
+TiKV has an optimistic transaction model, which means the client will buffer all the writes within a transaction, and when the client calls commit function, the writes will be packed and sent to the servers. If there is no conflict, the writes which are the key-value pairs with specific version are written into the database, and can be read by other transactions.
+
+## Can I use TiKV as a key-value store?
+
+Yes. For now, we have a [TiKV driver for Go](https://github.com/pingcap/tidb/tree/master/store/tikv). You are welcome to develop drivers in other languages!
+
+## How does TiKV compare to traditional relational databases like Oracle or MySQL?
+
+TiKV scores in horizontal scalability while still maintains the traditional relational database features. You can easily increase the capacity or balance the load by adding more machines.
+
+## How does TiKV compare to NoSQL databases like Cassandra, HBase, or MongoDB?
+
+TiKV is as scalable as NoSQL databases but features in the usability and functionality of traditional SQL databases, such as SQL syntax and consistent distributed transactions.
+
+## What is the recommended number of replicas in the TiKV cluster? Is it better to keep the minimum number for high availability?
+
+Use 3 replicas for testing. If you increase the number of replicas, the performance declines but it is more secure. Whether to configure more replicas depends on the specific business needs.
+
+## If a node is down, will the service be affected? How long?
+
+TiKV uses Raft to synchronize data among multiple replicas and guarantees the strong consistency of data. If one replica goes wrong, the other replicas can guarantee data security. The default number of replicas in each Region is 3. Based on the Raft protocol, a leader is elected in each Region, and if a single leader fails, a follower is soon elected as Region leader after a maximum of 2 * lease time (lease time is 10 seconds).
```

Comment:
I don't think this answer actually answers the question, does it? It just says "Yes your data is secure" it doesn't say that the service should remain available (potentially after a short reelection time)

Reply:
Right! @youjiali1995 @hicqu  Can you help "re-answer" this question?

Reply:
Besides, we usually say "safety" instead of "security". "security" means your data won't be compromised by hackers, "safety" means your data won't suffer from corruption or loss.